pages/status.tsx

```diff
@@ -146,6 +147,11 @@ const Status: FC = () => {
                 checkAgainText={t('check-again')}
                 checkStatusResponse={checkStatusResponse}
               />
+              <div className="mt-10">
+                <Trans i18nKey={'feedback-link'}>
```

Comment:
Add `common` namespace to the key as `'common:feedback-link'`. Apply it in email page as well.